src/pages/categories.js

```diff
@@ -1,28 +1,25 @@
-import React from "react"
-import PropTypes from "prop-types"
-
-// Utilities
-import kebabCase from "lodash/kebabCase"
-
-// Components
-import { Link, graphql } from "gatsby"
-
-const CategoriesPage = ({ data }) => (
-  <>
-    <div>
-      <h1>CategoriesPage</h1>
-      <ul>
-        {data.allMdx.group.map(category => (
-          <li key={category.fieldValue}>
-            <Link to={`/categories/${kebabCase(category.fieldValue)}/`}>
-              {category.fieldValue} ({category.totalCount})
-            </Link>
-          </li>
-        ))}
-      </ul>
-    </div>
-  </>
-)
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { Link, graphql } from "gatsby";
+
+import { Pager } from '../components/pager';
+import { Card } from "../components/card";
+
+import * as styles from "./categories.module.scss";
```

Comment:
```suggestion
import styles from "./categories.module.scss";
```

Reply:
If I use the styles in this way, Gtasby throws an error: 
![imagen](https://github.com/xmartlabs/xl-blog/assets/107500715/e23b5db1-1fce-4a51-87d8-3a16e5c54afb)

Reply:
damn, gatsby!. no worries, leave it as it is

Reply:
Done